internal/jobobject/jobobject.go

```diff
@@ -546,3 +548,62 @@ func (job *JobObject) PromoteToSilo() error {
 func (job *JobObject) isSilo() bool {
 	return atomic.LoadUint32(&job.isAppSilo) == 1
 }
+
+// PrivateWorkingSet returns the private working set size for the job. This is calculated by adding up the
+// private working set for every process running in the job.
+func (job *JobObject) PrivateWorkingSet() (uint64, error) {
```

Comment:
Not QueryPrivateWorkingSet?

Reply:
Oh good call, follows the rest of the methods.

Reply:
@msscotb Changed